### PR TITLE
Fix habitat package

### PR DESCRIPTION
**Pull request by jaym**
This has a runtime dependency on bundler, ffi, and postgresql

### CODE REVIEW

File: habitat/plan.sh [hunk 1]
```diff
@@ -9,19 +9,19 @@ pkg_lib_dirs=(lib)
...
   core/make
   core/openssl
-  core/postgresql-client
   core/rsync
 )
 pkg_deps=(
   core/coreutils
   core/gcc
   core/ruby
+  core/bundler
```
Comment (jaym) [inline habitat/plan.sh:22]: we should probably look into this a bit more, we might not need bundler if we give it the right flags

Comment (jaym) [reply habitat/plan.sh:22]: there's a `--standalone` flag

Comment (tas50) [reply habitat/plan.sh:22]: If we're using ruby 2.6+ it comes with bundler built in

Comment (jaym) [reply habitat/plan.sh:22]: or hab package is 2.5.7 and seems to have it

**Pull request merged**